core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -306,6 +314,58 @@ oppia.controller('StateResponses', [
         return choiceIndices.every(function(choiceIndex) {
           return handledAnswersArray.indexOf(choiceIndex) !== -1;
         });
+      } else if (interactionId === 'ItemSelectionInput' &&
+                  maxSelectionCount === 1) {
+        var numChoices = $scope.getAnswerChoices().length;
+        choiceRules = [];
```

Comment:
I'm confused about what each element in choiceRules represents. Please could you explain?